app/dashboard/src/modals/UpsertSecretModal.tsx

```diff
@@ -32,7 +32,9 @@ export default function UpsertSecretModal(props: UpsertSecretModalProps) {
       <Form
         testId="upsert-secret-modal"
         method="dialog"
-        schema={(z) => z.object({ name: z.string().min(1), value: z.string() })}
+        schema={(z) =>
```

Comment:
nit: for arbitrary errors we can adjust messages here: https://github.com/enso-org/enso/blob/56d1b00b6911138b7233bf336f17d320f406ef8a/app/dashboard/src/components/AriaComponents/Form/components/useForm.ts#L86

Reply:
is there a way to detect `string().min(1)` though?